test/functional/wallet_keypool_topup.py

```diff
@@ -25,70 +17,66 @@ def add_options(self, parser):
 
     def set_test_params(self):
         self.setup_clean_chain = True
-        self.num_nodes = 4
-        self.extra_args = [[], ['-keypool=100'], ['-keypool=100'], ['-keypool=100']]
+        self.num_nodes = 1
+        self.extra_args = [['-keypool=100']]
 
     def skip_test_if_missing_module(self):
         self.skip_if_no_wallet()
 
     def run_test(self):
-        wallet_path = self.nodes[1].wallets_path / self.default_wallet_name / self.wallet_data_filename
-        wallet_backup_path = self.nodes[1].datadir_path / "wallet.bak"
-        self.generate(self.nodes[0], COINBASE_MATURITY + 1)
-
-        self.log.info("Make backup of wallet")
-        self.stop_node(1)
-        shutil.copyfile(wallet_path, wallet_backup_path)
-        self.start_node(1, self.extra_args[1])
-        self.connect_nodes(0, 1)
-        self.connect_nodes(0, 2)
-        self.connect_nodes(0, 3)
-
-        for i, output_type in enumerate(["legacy", "p2sh-segwit", "bech32"]):
-
+        node = self.nodes[0]
+        self.generate(node, COINBASE_MATURITY + 1)
```

Comment:
Can remove this line by dropping the `setup_clean_chain` set line:
```diff
diff --git a/test/functional/wallet_keypool_topup.py b/test/functional/wallet_keypool_topup.py
--- a/test/functional/wallet_keypool_topup.py	(revision 6755e60fffd7748e7cbc16c9b18d84e62f26c697)
+++ b/test/functional/wallet_keypool_topup.py	(date 1710017695860)
@@ -16,7 +16,6 @@
         self.add_wallet_options(parser)
 
     def set_test_params(self):
-        self.setup_clean_chain = True
         self.num_nodes = 1
         self.extra_args = [['-keypool=100']]
 
@@ -25,7 +24,6 @@
 
     def run_test(self):
         node = self.nodes[0]
-        self.generate(node, COINBASE_MATURITY + 1)
 
         output_types = ["legacy", "p2sh-segwit", "bech32"]
         if self.options.descriptors:
```

Reply:
Done